core/src/main/java/org/apache/carbondata/core/metadata/datatype/Field.java

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package org.apache.carbondata.sdk.file;
+package org.apache.carbondata.core.metadata.datatype;
```

Comment:
Why move this package? Originally, this class is intended stay in SDK module so that user only uses API in carbondata-sdk module instead of calling functions in carbondata-core module

Reply:
Actually Field class already has the logic to parse complex child types, therefore instead of duplicating the code its better to move this class to core and use in both flows.

Reply:
SDK guide need to update, many examples in guide have import statement of Field